src/containers/BookListContainer.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+import BookList from '../components/BookList';
+import { removeBook, changeFilter } from '../actions/index';
+
+const mapStateToProps = state => {
+  const { books } = state;
+  const filteredBooks = books.filter(book => book.category === state.filter);
+  if (state.filter !== '') return { books: filteredBooks };
+  return { books };
+};
+
+const mapDispatchToProps = dispatch => ({
```

Comment:
Hello `mapDispatchToProps`. haha didn't expect to see you here :D Kindly see all my comments related to `mapDispatchToProps` definitions in your previous PR